test/acceptance/behave/components/ngsiv2/entities/list_entities/list_entities_qp_II.feature

```diff
@@ -0,0 +1,1289 @@
+# -*- coding: utf-8 -*-
```

Comment:
Same here...

Ideally, each file should have some internal coherence in order to chose a good name, based on the kind of test cases it includes. If this is not possible (due to the same file mixes test cases very different among them) then use `_part1`, `_part2`, etc.